src/libos/crates/async-rt/src/scheduler/local_scheduler/runqueues.rs

```diff
@@ -48,11 +51,22 @@ impl<E: SchedEntity> RunQueues<E> {
             unsafe { mem::transmute(run_queues) }
         };
         Self {
+            next_entity,
             run_queues,
             nonempty_mask: 0,
         }
     }
 
+    pub fn enqueue_waked_task(&mut self, entity: Arc<E>) {
```

Comment:
Choose more better names, indicating that tasks will be prioritized for dequeue.

Reply:
Done.